src/core/public/apm_system.ts

```diff
@@ -73,4 +78,52 @@ export class ApmSystem {
       }
     });
   }
+
+  /**
+   * Adds a filter to the APM configuration for normalizing transactions of the 'http-request' type to remove the
+   * hostname, protocol, port, and base path. Allows for coorelating data cross different deployments.
+   */
+  private addHttpRequestNormalization(apm: ApmBase) {
+    apm.addFilter((payload) => {
```

Comment:
There is a better API to do this using the observe API where you can filter by specific transaction. 

https://www.elastic.co/guide/en/apm/agent/rum-js/current/agent-api.html#observe

```js
apm.observe('transaction:end', (transaction) => {
  if (transaction.type === 'custom') {
    transaction.name = window.document.title
  }
})
```